sort.py

```diff
@@ -0,0 +1,14 @@
+import random
+
+def sort_add_num(list_, num):
```

Comment:
```suggestion
def sort_add_num(list_ = None:, num=0):
```
デフォルト引数を設定して下さい。
please set degfault arguments

Reply:
OK